CMakeLists.txt

```diff
@@ -1245,3 +1273,14 @@ if(CONFIG_X86 AND CONFIG_USERSPACE AND NOT CONFIG_X86_NO_MELTDOWN)
       does not contain a fix for this issue."
 )
 endif()
+
+if(CONFIG_APP_SHARED_MEM)
```

Comment:
Where exactly are you facing the linker issue in prebuilt? 
As of now I am able to build and run without any issues when I disable this section.